lib/myarray.rb

```diff
@@ -0,0 +1,22 @@
+class My_array
+
+
+
+ def my_inject array, &my_block
+   acc = array
+   acc.each do |num|
+   result = yield(acc, num)
+  end
```

Comment:
Lint/BlockAlignment: `end` at 9, 2 is not aligned with `acc.each do |num|` at 7, 3